src/main/java/com/google/api/codegen/transformer/py/PythonImportSectionTransformer.java

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.py;
+
+import com.google.api.codegen.metacode.InitCodeNode;
+import com.google.api.codegen.transformer.ImportSectionTransformer;
+import com.google.api.codegen.transformer.MethodTransformerContext;
+import com.google.api.codegen.transformer.ModelTypeTable;
+import com.google.api.codegen.transformer.SurfaceTransformerContext;
+import com.google.api.codegen.viewmodel.ImportFileView;
+import com.google.api.codegen.viewmodel.ImportSectionView;
+import com.google.api.codegen.viewmodel.ImportTypeView;
+import com.google.api.tools.framework.model.TypeRef;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class PythonImportSectionTransformer implements ImportSectionTransformer {
```

Comment:
(Throughout this file)

Prefer to chain builder methods, i.e.,

```
Foo foo = Foo.newBuilder()
    .xxx("a")
    .yyy("b")
    .zzz("c")
    .build();
```

instead of

```
Foo.Builder foo = Foo.newBuilder();
foo.xxx("a");
foo.yyy("b");
foo.zzz("c");
Foo result = foo.build();
```